src/components/Auth/SignupPage/Steps/MobileExternalDetailStep/index.tsx

```diff
@@ -152,6 +160,7 @@ function MobileExternalDetailStep({ onNext }: MobileExternalDetailStepProps) {
                 {...field}
                 placeholder="5~13자리로 입력해 주세요."
                 isButton
+                onChange={(e) => {handleLoginIdChange(e, field);}}
```

Comment:
밑의 코드 포함해서 아래와 같이는 안되나요?

```suggestion
                onChange={(e) => handleLoginIdChange(e, field)}
```

Reply:
자동완성으로 적용하다가 놓친 것 같네요.. 다음부터는 잘 확인하도록 하겠습니다
같은 코드 부분들 수정했습니다!
[d3771b0](https://github.com/BCSDLab/KOIN_WEB_RECODE/pull/1021/commits/d3771b0d863ae138ee2ecf9852f41a60f865105e)